ikenna-portfolio/portfolio/src/main/java/com/google/sps/data/Project.java

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/**
+* The Project class stores information that represents how the projects container
+* will look like.
+* 
+*  
+*/
+public class Project {
+  private final String name;
+  private final String imageHTML;
+  private final String desc;
+  private final String links;
+  
+  public Project(String name, String img1, String img2, String img3, String desc, String links) {
+    this(name, img1, img2, img3, desc, links, 3);
+  }
+
+  public Project(String name, String img1, String img2, String img3, String desc, String links, int sections) {
+    this.name = name;
+    this.imageHTML = getImageHTML(sections, new String[] {img1, img2, img3});
+    this.desc = desc;
+    this.links = links;
+  }
+
+  private String getImageHTML(int sections, String[] src) {
+    String html = "";
+    if (sections == 3) {
```

Comment:
done-ish; I'm not sure how to it otherwise. The reasoning for the difference in html is because if I use three panels for images when I only have one then broken images show up and make it look sorta ugly. Do you have any recommendation for this?

Reply:
Ideally you could handle this more gracefully since it's possible that the UI will have a variety of images in the future, not just 3 or 1. Maybe in the meantime, you could hardcode 3 so that it doesn't seem as random- like MAX_IMAGE_COUNT or something.

Reply:
done